### ISSUE TITLE
New Riverpod providers?

### ISSUE THREAD

**Issue by franzaps** [flutterdata/flutter_data#276]
Been using this library in many projects, it's great thanks!

Will you use the latest format for Riverpod providers or integrate better with them?

**Comment by frank06** [comment]
Yes. Migrating to those is part of the plan for 2.x , but it will come a bit later on
